mod/ArchConsoleManager.cs

```diff
@@ -187,23 +195,31 @@ public void AddText(string text, bool skipGameplayConsole = false, AudioType sou
             // If the consoles haven't been created yet, then adding to history is all we want to do for now.
             if (pauseConsoleText == null) return;
 
-            string consoleText = pauseConsoleText.text;
-            if (consoleText == "")
+            // bufferedText = pauseConsoleText.text;
```

Comment:
dead code?

Reply:
Oops, I knew there was a comment I forgot to remove.